process/interface.go

```diff
@@ -383,6 +383,8 @@ type EpochStartTriggerHandler interface {
 	SetProcessed(header data.HeaderHandler)
 	Revert(round uint64)
 	EpochStartMetaHdrHash() []byte
+	GetSavedStateKey() []byte
```

Comment:
It seems a little odd that there is a LoadState and there is not a StoreState, but I see that the save is called from multiple places and it is private.

Reply:
the load is called by the bootstrapper, since it has the keys for the state it wants to boostrap from.
The store is done by each component since it is responsible with the state it needs to persist and when it is relevant to be persisted. 

I can rename this interface function to InitFromSavedStateKey which can then call a private loadState if you think it needs it.

Reply:
it's ok!